frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_v3/appflowy_mobile_toolbar.dart

```diff
@@ -408,6 +415,7 @@ class _MobileToolbarState extends State<_MobileToolbar>
                 );
               }
             }
+            if (keyboardHeight > 0) _maxKeyboardHeight = keyboardHeight;
```

Comment:
The name `_maxKeyboardHeight` is not appropriate in this case. _maxKeyboardHeight should be a value that only stores the maximum height.

Reply:
I've changed `_maxKeyboardHeight ` to `_globalCachedKeyboardHeight`, `_maxKeyboardHeight` is misleading.